chain/arweave/src/runtime/generated.rs

```diff
@@ -0,0 +1,129 @@
+use graph::runtime::{AscIndexId, AscPtr, AscType, DeterministicHostError, IndexForAscTypeId};
+use graph::semver::Version;
+use graph_runtime_derive::AscType;
+use graph_runtime_wasm::asc_abi::class::{Array, AscString, Uint8Array};
+
+#[repr(C)]
+#[derive(AscType, Default)]
+pub struct AscBlock {
+    // pub ver: u32,
+    pub timestamp: u64,
+    pub last_retarget: u64,
+    pub height: u64,
```

Comment:
Have you validated that the `u64`s here don't cause padding issues because of [this bug](https://github.com/graphprotocol/graph-node/issues/3011)?

Reply:
no padding issues, see https://github.com/chainSafe/subgraph-arweave-example